internal/db/models.go

```diff
@@ -123,6 +123,7 @@ type ProjectResource struct {
 	MinQuotaFromBackend     *uint64             `db:"min_quota_from_backend"`
 	MaxQuotaFromBackend     *uint64             `db:"max_quota_from_backend"`
 	MaxQuotaFromAdmin       *uint64             `db:"max_quota_from_admin"`
+	MaxQuotaFromProject     *uint64             `db:"max_quota_from_project"`
```

Comment:
It's becoming evident to me that the original `max_quota_from_admin` naming was flawed and should be fixed (Postgres has `ALTER TABLE ... RENAME` for this purpose). "From admin" vs. "from project" is not a good contrast because there also exists admins on the project level (but "from admin" refers to admins on the domain level). I suggest:
```suggestion
	MaxQuotaFromOutsideAdmin *uint64             `db:"max_quota_from_outside_admin"`
	MaxQuotaFromLocalAdmin   *uint64             `db:"max_quota_from_local_admin"`
```

Reply:
I updated the naming scheme accordingly